jenkins-scripts/lib/homebrew_remove_dependent_bottles.bash

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash -x
+
+# Knowing Script dir beware of symlink
+[[ -L ${0} ]] && SCRIPT_LIBDIR=$(readlink ${0}) || SCRIPT_LIBDIR=${0}
+SCRIPT_LIBDIR="${SCRIPT_LIBDIR%/*}"
+
+echo '# BEGIN SECTION: check variables'
+if [ -z "${BROKEN_FORMULA}" ]; then
+  echo BROKEN_FORMULA not specified
+  exit -1
+fi
+if [ -z "${BOTTLE_TAG}" ]; then
+  echo BOTTLE_TAG not specified
+  exit -1
+fi
+echo '# END SECTION'
+
+PULL_REQUEST_HEAD_REPO=git@github.com:osrfbuild/homebrew-simulation.git
```

Comment:
Seems not to be used in this file, is to be exported?

Reply:
it's required by `jenkins-scripts/lib/_homebrew_github_setup.bash`, which is called shortly after